components/generic/CiteAttribution.vue

```diff
@@ -1,5 +1,61 @@
 <template>
-  <cite>
+  <cite v-if="extended">
+    <p v-if="name !== null">{{ $t('exhibitions.attribution.title') }}
+      <SmartLink
+        v-if="url !== null"
+        :destination="url"
+      >
+        {{ name }}
+      </SmartLink>
+      <span v-else>
+        {{ name }}
+      </span>
+    </p>
+    <p v-if="creator !== null">{{ $t('exhibitions.attribution.creator') }}
```

Comment:
CiteAttribution should not need to know anything about exhibitions. From the perspective of this component, this is a stylistic variant. Rename accordingly.